idl/response/foxsi_get_effective_area.pro

```diff
@@ -41,7 +41,8 @@ FUNCTION foxsi_get_effective_area, ENERGY_ARR = energy_arr, PLOT = plot, $
     ENDELSE
     detector = foxsi_get_detector_efficiency(energy_arr = energy_arr)
     blanket = foxsi_get_blanket_transmission(energy_arr = energy_arr)
-    shutter = foxsi_get_shutter_transmission(shutter_state, energy_arr = energy_arr)
+    shutter = foxsi_get_shutter_transmission(shutter_state, energy_arr = energy_arr, $
+    					shutter_thick_mm = shutter_thick_mm, _extra=_extra)

```

Comment:
Did you mean to include/leave the `_extra` keyword?  It hasn't been added to the call for `foxsi_get_effective_area()`, so it currently doesn't do anything (unless I'm missing something).